src/main/java/com/jcabi/s3/Bucket.java

```diff
@@ -120,7 +120,10 @@ void remove(@NotNull(message = "S3 key can't be NULL") String key)
          */
         private final transient Bucket origin;
         /**
-         * Prefix.
+         * Prefix is a string which is used.
```

Comment:
I don't get what you're saying in this Javadoc description. Maybe it's better if you can provide an example.